### ISSUE TITLE
Handle when Simple Summary has multiple paragraphs

### ISSUE THREAD

**Issue by raykyri** [canvasxyz/metropolis#67]

**Comment by rjwebb** [comment]
Ok, the solution to this is pretty simple in the happy case (most FIPs and PRs). We can look for the first heading called "Simple Summary", then iterate over the subsequent nodes until we find another heading that as large or larger (i.e. if Simple Summary is a H2 then we iterate until we find the next H1 or H2 or title). Draft is here: https://github.com/canvasxyz/metropolis/pull/70

One edge case that I've found is that some FIPs don't have a "Simple Summary" heading, I'm guessing these were written before this was added to the process, e.g. https://github.com/filecoin-project/FIPs/blob/master/FIPS/fip-0001.md , https://github.com/filecoin-project/FIPs/blob/master/FIPS/fip-0018.md https://github.com/filecoin-project/FIPs/blob/master/FIPS/fip-0080.md I think in these cases we should extract the content below the "Summary" heading instead.
